lib/modules/modules.go

```diff
@@ -174,6 +187,46 @@ func (f Features) ToProto() *proto.Features {
 	}
 }
 
+// ProductType is the type of product.
+type ProductType int32
+
+const (
+	ProductTypeUnknown ProductType = 0
+	// ProductTypeTeam is Teleport ProductTypeTeam product.
+	ProductTypeTeam ProductType = 1
+	// ProductTypeEUB is Teleport Enterprise Usage Based product.
+	ProductTypeEUB ProductType = 2
+)
+
+// IsLegacy describes legacy product that existed before EUB product
+// was introduced.
+func (f Features) IsLegacy() bool {
+	return !f.IsUsageBasedBilling
+}
+
+func (f Features) IGSEnabled() bool {
+	return f.IdentityGovernanceSecurity
+}
+
+func GetAccessListFeatureConfig() AccessListFeature {
+	return AccessListFeature{
+		CreateLimit: 1,
+	}
+}
+
+func GetAccessRequestFeatureConfig() AccessRequestsFeature {
+	return AccessRequestsFeature{
+		MonthlyRequestLimit: 5,
+	}
+}
+
+func GetDeviceTrustFeatureConfig() DeviceTrustFeature {
```

Comment:
Could we add a godoc?

Could we rename this so it's clear that these are the default config for usage-based enterprise plans? Alternatively, should these live in e/ instead?

Same for similar methods.

Reply:
moved it to enterprise